tests/vmi_multus_test.go

```diff
@@ -874,19 +874,25 @@ var _ = Describe("[Serial]SRIOV", func() {
 			Expect(configureInterfaceStaticIPByMAC(vmi1, mac1.String(), cidrA)).To(Succeed())
 			Expect(configureInterfaceStaticIPByMAC(vmi2, mac2.String(), cidrB)).To(Succeed())
 
-			// now check ICMP goes both ways
-			Expect(libnet.PingFromVMConsole(vmi1, cidrToIP(cidrB))).To(Succeed())
-			Expect(libnet.PingFromVMConsole(vmi2, cidrToIP(cidrA))).To(Succeed())
+			return vmi1, vmi2
 		}
 
 		It("[test_id:3956]should connect to another machine with sriov interface over IPv4", func() {
+			cidrA := "192.168.1.1/24"
+			cidrB := "192.168.1.2/24"
+			vmi1, vmi2 := createSriovVMs(cidrA, cidrB)
 			Skip("Skip until https://github.com/kubevirt/kubevirt/issues/3774 fixed")
```

Comment:
You mentioned this in the commit message: `in order to skip as late as possible.`
But you did not explained why you do this. I do not understand why there is a need to create the VMI/s and only then skip the ping.

Reply:
Because i added the e2e code that checks for the network name in the createSriovVMs.
The skip was first line in this function, so i moved it to be just before the ping since they are the reason for the skip,
and then it would check all, but the pings, including the e2e logic i added for the network name.

Reply:
will be fixed as part of https://github.com/kubevirt/kubevirt/pull/4335#discussion_r519874886
since i will move the logic of checking network name to test 3985

but even if i fix it, its good to have the skip just on the part that should be skipped, so we check the rest of the code, unless we dont care for it until the PR that fix it would be merged